connection_test.go

```diff
@@ -499,13 +497,19 @@ func TestConcurrentReadOnParams(t *testing.T) {
 				if rows == nil {
 					continue
 				}
+				rows.Next()
+				var tableName string
+				err = rows.Scan(&tableName)
+				if err != nil {
+					t.Error(err)
+				}
 				_ = rows.Close()
 			}
 			wg.Done()
 		}()
 	}
 	wg.Wait()
-	defer db.Close()
+	db.Close()
```

Comment:
do we need that since we deferred db.Close in line 483?

Reply:
Good catch!